presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -852,4 +883,28 @@ private void assertQueryResultCount(String sql, int expectedResultCount)
     {
         assertEquals(getQueryRunner().execute(sql).getRowCount(), expectedResultCount);
     }
+
+    private void assertBigEndian32Conversion(Integer value)
```

Comment:
Why do we need these two functions? Why can't we write ` assertQuery(...)` like all other tests in this file?
It's hard to read this style.

Reply:
Apologies, I was initially asked to refactor the tests this way, let me change that back to assertQuery.

Reply:
@majetideepak : I had requested that re-factor since it made it easier to look at the individual values being tested with the functions. But this is fine also.

Thanks @isadikov for refactoring.